grails-app/domain/org/pih/warehouse/core/Location.groovy

```diff
@@ -339,6 +346,13 @@ class Location implements Comparable<Location>, java.io.Serializable {
         return active ? LocationStatus.ENABLED : LocationStatus.DISABLED
     }
 
+    List<User> getUsersWithRole(RoleType roleType) {
+        withNewSession {
+            List<Role> roles = Role.findAllByRoleType(roleType)
```

Comment:
decided to use `findAll` to be "safe", since hypothetically a user can create a few roles with the roleType of REQUEST_APPROVER